src/components/Footer/Footer.jsx

```diff
@@ -1,39 +1,30 @@
-// React Imports
 import React from 'react';
-// Material UI Imports
 import Box from '@mui/material/Box';
 import Divider from '@mui/material/Divider';
 import Stack from '@mui/material/Stack';
 import useMediaQuery from '@mui/material/useMediaQuery';
 import { useTheme } from '@mui/material/styles';
-// Component Imports
 import RenderCallToActionSection from './RenderCallToActionSection';
 import RenderCompanyInfoSection from './RenderCompanyInfoSection';
 import RenderCopyrightAndLinksSection from './RenderCopyrightAndLinksSection';
 
-/**
- * Footer Component - Generates the responsive footer used in PASS
- *
- * @memberof Footer
- * @name Footer
- * @returns {React.JSX.Element} The Footer component
- */
```

Comment:
We need to keep all of these JSDocs. We're using them for the whole project and it would be unusual to remove them just for these.